header.css

```
@@ -0,0 +1,25 @@
+/* header */
+header {
+    height: 4rem;
+}
+/* search-bar */
+.search-bar {
+    max-width: 36em;
+}
+
+.search-bar {
+    outline: none;
+    border: none;
+}
```

@Sakshi-Simform Can consolidate these properties into single class selector?
```suggestion
/* search-bar */
.search-bar {
    max-width: 36em;
    outline: none;
    border: none;
}
```

---

merged the properties in one class https://github.com/Sakshi-Simform/flipkart-ui/pull/2/commits/643b93e1245abab4d2f33515e30df6d6d05a5625